### PR TITLE
Update version and date in Changelog

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
During the release 5.2.1 we forgot to change the version in Changelog from 5.2.0.

# Solution
Update the version and date of the release

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://674-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [Next version]
 
-## [5.2.0] - 2019-05-29
+## [5.2.1] - 2019-05-30
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: Why is the version different here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -412,7 +412,7 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 
 [next-version]:
-[5.2.0-rc.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.0
+[5.2.0]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.1
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: This should be
```
[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/5.2.1...development
[5.2.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.1
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -412,7 +412,7 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
...
 [5.1.0]: https://github.com/onfido/onfido-sdk-ui/compare/5.0.1...5.1.0
 [5.0.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.0.0...5.0.1
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: `[5.0.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.0.0...5.0.1`
This is link is wrong and should be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -412,7 +412,7 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 
 [next-version]:
-[5.2.0-rc.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.0
+[5.2.0]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.1
 [5.1.0]: https://github.com/onfido/onfido-sdk-ui/compare/5.0.1...5.1.0
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: This one should be `[5.1.0]: https://github.com/onfido/onfido-sdk-ui/compare/5.0.0...5.1.0`

File: CHANGELOG.md (hunk 1 above)
Comment (bartzet) [reply CHANGELOG.md]: https://github.com/onfido/onfido-sdk-ui/pull/673#discussion_r289322875

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-674-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://674-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**